drivers/gps/nrf9160_gps/nrf9160_gps.c

```diff
@@ -480,11 +485,6 @@ static int parse_cfg(struct gps_config *cfg_src,
 			return -EINVAL;
 		}
 
-		if (cfg_src->timeout >= cfg_src->interval) {
```

Comment:
Might split this change into separate commit

Reply:
done